src/rulesets/api-lifeycle.ts

```diff
@@ -30,6 +30,9 @@ export const rules = {
       "not change unless it was wip",
       (before, after, context, docs) => {
         if (!before && !after) return;
+        // Allow for creating new resource versions with any stability.
+        // No `before` means this is a new resource
+        if (!before) return;
         if (before === "wip") return;
         expect(before).to.equal(after);
```

Comment:
What about the case where `after` is removed, but it's sunset-eligible to be removed?

Reply:
I'm currently working on handling sunset rules in another PR and another rule function. Do you think that will cover this scenario?

Reply:
I think it should. So perhaps here, `if (!before || !after) return;` and let the sunset rule catch any premature removal of `after`.

Reply:
Ah, you're right there. I'll change that.